Dockerfile

```diff
@@ -6,7 +6,8 @@ ARG PREFIX=/usr/local
 RUN dpkg --add-architecture i386
 
 # Add rocm repository
-RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://repo.radeon.com/rocm/apt/4.5/ ubuntu main > /etc/apt/sources.list.d/rocm.list'
+RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://repo.radeon.com/rocm/apt/5.0.2/ ubuntu main > /etc/apt/sources.list.d/rocm.list'
+RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://repo.radeon.com/amdgpu/21.50.2/ubuntu bionic main > /etc/apt/sources.list.d/amdgpu.list'
```

Comment:
Is this needed? We only install the userspace packages in the docker, and I think this doesn't have any userspace packages.

Reply:
removed the amdgpu installer repo